www/source/layouts/_footer.slim

```diff
@@ -6,10 +6,10 @@ footer#main-footer.dark-bg
 
     .row.align.columns
       .strict-center
-        a.button.btn-lg.btn-purple.shadow-dark.margin-right-xs.mobile-row-btn href="/downloads"
+        a.button.btn-lg.btn-purple.shadow-dark.margin-right-xs.mobile-row-btn href="https://downloads.chef.io/" target="_blank"
           i.fa.fa-cloud-download
           span download InSpec 4
-        a.button.btn-lg.btn-white-o.shadow-dark.mobile-row-btn href="/tutorials" browse tutorials
+        a.button.btn-lg.btn-white-o.shadow-dark.mobile-row-btn href="https://learn.chef.io/" target="_blank" browse tutorials
```

Comment:
There's also currently a link to `/tutorials` at https://www.inspec.io/docs/

I think that page can be deleted because it no longer links to valid Chef Rally URLs